data/helpers.d/network

```diff
@@ -17,7 +17,7 @@ ynh_find_port () {
 	ynh_handle_getopts_args "$@"
 
 	test -n "$port" || ynh_die --message="The argument of ynh_find_port must be a valid port."
-	while netcat -z 127.0.0.1 $port       # Check if the port is free
+	while netstat -ltu | grep -q -w :$port       # Check if the port is free
```

Comment:
```suggestion
	while netstat -nltu | grep -q -w :$port       # Check if the port is free
```

Gotta add `-n`, otherwise it tries to replace well-known ports like `443` to `https` or `5222` to `xmpp-client` and so on

Reply:
Yes indeed!